generators/generator-bot-enterprise-calendar/README.md

```diff
@@ -1,47 +1,39 @@
 # @microsoft/generator-bot-enterprise-calendar [![NPM version][npm-image]][npm-url]
 
-This template creates a bot configured to manage Office 365 calendars using the Microsoft Graph API. **[Learn more](https://aka.ms/EnterpriseCalendarBot)**
```

Comment:
The README.md in the repo root refers to M365. I would assume we should use consistent terminology. Office 365 is a little more friendly to those less familiar with acronyms, so my vote would be to change the reference in the root README.md.

Reply:
I standardized on Office 365 across any readme that referenced M365